zeq/types.h

```diff
@@ -32,7 +32,9 @@ namespace zeq
 class Event;
 class Publisher;
 class Subscriber;
+struct EventDescriptor;
 
+typedef std::vector< ::zeq::EventDescriptor > EventDescriptorsVector;
```

Comment:
Ok I will change that but I think, in general, EventDescriptorsVector is a more clear name.